controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/CommitRequestHandler.java

```diff
@@ -106,12 +113,13 @@ public CommitRequestHandler(final StreamMetadataStore streamMetadataStore,
                         } else {
                             log.error("Exception while attempting to commit transaction on stream {}/{}", scope, stream, e);
                         }
+                        transactionMetrics.commitTransactionFailed(scope, stream);
                         future.completeExceptionally(cause);
                     } else {
                         if (r >= 0) {
                             log.info("Successfully committed transactions on epoch {} on stream {}/{}", r, scope, stream);
                         } else {
-                            log.info("No transactions found in committing state on stream {}/{}", r, scope, stream);
+                            log.info("No transactions found in committing state {} on stream {}/{}", r, scope, stream);
```

Comment:
please replace this with
`                             log.info("No transactions found in committing state on stream {}/{}", scope, stream); `

Reply:
Fixed.